docs/sources/release-notes/v2.3.md

```diff
@@ -29,6 +29,11 @@ For the complete list of changes, see the [Changelog](https://github.com/grafana
 - **Added `backfill` command to mimirtool to upload Prometheus TSDB blocks:**
   The `mimirtool backfill` command will make the process of back-filling Prometheus TSDB blocks, using an API available in the compactor, even easier which allows for a migration to Grafana Mimir while keeping your previous Prometheus TSDB data.
 
+- **Helm Helm Chart improvements:** The Mimir Helm chart is the best way to install Mimir on Kubernetes. As part of the Mimir 2.3 release, we’re also releasing version 3.1 of the Helm chart. Notable enhancements follow. For the full list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
```

Comment:
```suggestion
- **New Helm chart minor release:** The Mimir Helm chart is the best way to install Mimir on Kubernetes. As part of the Mimir 2.3 release, we’re also releasing version 3.1 of the Mimir Helm chart. Notable enhancements follow. For the full list of changes, see the [Helm chart changelog](https://github.com/grafana/mimir/tree/main/operations/helm/charts/mimir-distributed/CHANGELOG.md).
```

Reply:
I think in a few more releases we can stop saying "The Mimir Helm chart is the best way to install Mimir on Kubernetes.", but for now let's build the muscle in the community.